docs/topics/request-response.rst

```diff
@@ -488,7 +488,10 @@ fields with form data from :class:`Response` objects.
 
 .. _lxml.html forms: https://lxml.de/lxmlhtml.html#forms
 
-.. class:: FormRequest(url, [formdata, ...])
+.. class:: scrapy.http.request.form.FormRequest
+.. class:: scrapy.http.request.FormRequest
```

Comment:
This import path does not actually exist, only the other 3 do.

Reply:
Do you mean line 491?

Reply:
No, I mean line 492. `scrapy.http.request.form.FormRequest` [exists](https://github.com/scrapy/scrapy/blob/master/scrapy/http/request/form.py#L19), but `scrapy.http.request.FormRequest` [does not](https://github.com/scrapy/scrapy/blob/master/scrapy/http/request/__init__.py).

Reply:
I see. Great, thank you!